### DISCUSSION TITLE
NetBSD/pkgsrc package available

### DISCUSSION THREAD

**Discussion by 0323pin** [arxanas/git-branchless#741] [General]
Hi,

I've just packaged and merged `git-branchless` into pkgsrc main branch, http://mail-index.netbsd.org/pkgsrc-changes/2023/01/12/msg267004.html

Regards,
pin

**Comment by arxanas** [comment]
Thanks! I've just added a badge for package managers at https://github.com/arxanas/git-branchless/wiki/Installation#installation. Do you know if the NetBSD package will appear there/on Repology at https://repology.org/project/git-branchless/versions?

**Comment by 0323pin** [comment]
Thanks!

> Do you know if the NetBSD package will appear there/on Repology at https://repology.org/project/git-branchless/versions?

It's already there. The thing is that Repology doesn't name it NetBSD but, pkgsrc current ;)
